spec/services/sign_in/access_token_jwt_encoder_spec.rb

```diff
@@ -45,6 +46,27 @@
         expect(decoded_jwt.jti).to eq expected_jti
         expect(decoded_jwt.aud).to eq expected_aud
       end
+
+      context 'expected user_attributes on encoded access token' do
+        context 'when there are no user attributes on the correlated ClientConfig access_token_attributes' do
+          it 'does not include user attributes on the encoded access token' do
+            decoded_jwt = OpenStruct.new(JWT.decode(subject, false, nil).first)
+            expect(decoded_jwt.user_attributes).to eq({})
+          end
+        end
+
+        context 'when there are one or more user attributes on the correlated ClientConfig access_token_attributes' do
+          let(:access_token_attributes) { %w[first_name last_name] }
+
+          it 'includes those attributes on the encoded access token' do
+            decoded_jwt = OpenStruct.new(JWT.decode(subject, false, nil).first)
+            serialized_attributes = decoded_jwt.user_attributes
+            expect(serialized_attributes['first_name']).to eq(access_token.user_attributes['first_name'])
```

Comment:
This is just nitpick, so feel free to do this or ignore it, but I prefer to put the expectation as a `let` if I can:
`let(:expected_first_name) { access_token.user_attributes['first_name'] }`